package.json

```diff
@@ -744,6 +755,9 @@
   },
   "dependencies": {
     "@azure/event-hubs": "^2.0.0",
+    "@types/body-parser": "^1.17.0",
+    "@types/express": "^4.17.0",
+    "@types/request-promise": "^4.1.44",
     "ajv": "^6.10.0",
     "axios": "^0.18.1",
     "azure-arm-eventhub": "^3.2.0",
```

Comment:
I find a warning in `azure-arm-eventhub`, do we need to update it?
```
#################################################################################
# Active development of this SDK has been moved to @azure/arm-eventhub package. #
#   This package is in maintenance mode and will be deprecated in July 2019.    #
#     Visit https://aka.ms/azure-sdk-for-js-migration for more information.     #
#################################################################################
```

Reply:
Treat this as a separate task. You can log a task in our sprint board for this.